src/olympia/addons/templates/addons/impala/details.html

```diff
@@ -143,21 +143,16 @@ <h3>{{ _('Enjoy this add-on?') }}</h3>
 
 {% block content %}
 
-{% if addon.type != amo.ADDON_PERSONA %}
-  {% set version = addon.current_version %}
-  {% if addon.type == amo.ADDON_STATICTHEME %}
-    {% set previews = version.previews.all() %}
-  {% else %}
-    {% set previews = addon.all_previews %}
-  {% endif %}
+{% set version = addon.current_version %}
```

Comment:
This `{% set ...` has moved out of the `{% if ...` block.  Not if that matters or is deliberate

Reply:
It was deliberate - there is already another one in a different block doing the same thing, not behind an `if`, so when getting rid of that `{% if ... %} {% else %} {% endif %}` I moved the `{% set %} call for consistency.